drivers/soundwire/bus.c

```diff
@@ -1032,13 +1021,8 @@ int sdw_bus_clk_stop(struct sdw_bus *bus)
 	 */
 	ret = sdw_bwrite_no_pm(bus, SDW_BROADCAST_DEV_NUM,
 			       SDW_SCP_CTRL, SDW_SCP_CTRL_CLK_STP_NOW);
-	if (ret < 0) {
-		if (ret == -ENODATA)
-			dev_dbg(bus->dev,
-				"ClockStopNow Broadcast msg ignored %d", ret);
-		else
-			dev_err(bus->dev,
-				"ClockStopNow Broadcast msg failed %d", ret);
+	if (ret < 0 && ret != -ENODATA) {
+		dev_err(bus->dev, "ClockStopNow Broadcast msg failed %d\n", ret);
```

Comment:
so this actually modifies behaviour: before `-ENODATA` would be returned, now we return 0 here.